src/lib/components/Swap/Summary/index.tsx

```diff
@@ -121,7 +126,8 @@ export function SummaryDialog({ trade, allowedSlippage, onConfirm }: SummaryDial
         <SummaryColumn gap={0.75} flex justify="center">
           <Summary input={inputAmount} output={outputAmount} usdc={true} />
           <ThemedText.Caption>
-            1 {inputCurrency.symbol} = {executionPrice?.toSignificant(6)} {outputCurrency.symbol}
+            {formatLocaleNumber({ number: 1, sigFigs: 1, locale: i18n.locale })} {inputCurrency.symbol} ={' '}
```

Comment:
Do you _need_ to use `{' '}` here? I'd avoid it if you don't need it.

Reply:
yeah, need it. html newlines dont count as spaces

Reply:
do you need a newline? could you put this all in one line?

Reply:
linting requires that this be two lines